lib/CachingReflector.php

```diff
@@ -75,7 +75,7 @@ public function getParamTypeHint(\ReflectionFunctionAbstract $function, \Reflect
             $paramCacheKey = self::CACHE_KEY_CLASSES . "{$lowClass}.{$lowMethod}.param-{$lowParam}";
         } else {
             $lowFunc = strtolower($function->name);
-            $paramCacheKey = ($lowFunc !== '{closure}')
```

Comment:
@bwoebi would it be possible to have this tagged in a release? I've got some code that is using funky param names to avoid it.